modules/client-configuration/pages/configuration-management.adoc

```diff
@@ -3,29 +3,26 @@
 
 You can use configuration files and channels to manage configuration for your clients, rather than configuring each client manually.
 
-[NOTE]
-====
-Some of the following features are available for traditional clients only.
-For features supported on {salt} clients, see the table below.
-====
-
 Configuration parameters are scripted and stored in configuration files.
 You can write configuration files directly using the {productname} {webui}, or you can upload or link to files that exist in other locations.
 
-Configuration files can be centrally managed, or locally managed.
+Configuration files can be centrally managed.
 Centrally managed configuration files are provided by global configuration channels and can be applied to any client subscribed to the {productname} Server.
+
+////
+// 2023-10, julio:
 Locally managed configuration files are used to override centrally managed configuration settings.
 They are especially useful for {productname} users who do not have configuration administration privileges, but need to make changes to the clients that they manage.
+////
 
 Configuration channels are used to organize configuration files.
 You can subscribe clients to configuration channels, and deploy configuration files as required.
 
 Configuration files are version-controlled, so you can add configuration settings, test them on your clients, and roll back to a previous revision as required.
 When you have created your configuration channels, you can also perform comparisons between various configuration files, and between revisions of the same configuration file.
 
-Configuration files can be centrally managed, or locally managed.
+Configuration files can be centrally managed.
```

Comment:
This line is the same as the one in line 9. Maybe it is not the purpose of this PR, but as there's been a modification here, would it be possible to remove it from one of the places? Actually even the beginning of the next sentence is also above...

Reply:
Done.

Reply:
thanks for input and help!